.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [ 20.x, 22.x, 24.x ]
+        node-version: [ 22.x, 24.x ]
```

Comment:
Hmm. If it isn't a gigantic lift, my preference would be to retain 20.x for the time being, even though it'll be EOL in 4mo. Obviously we want people to be on versions of Node that are actively maintained for security, but realistically folks aren't always quick to upgrade.